tests/Paginator/KeysetPaginatorTest.php

```diff
@@ -1168,4 +1168,84 @@ public function testGetPageToken(): void
 
         $this->assertSame($token, $paginator->getToken());
     }
+
+    public function testNextPage(): void
+    {
+        $dataSet = [['id' => 1], ['id' => 2], ['id' => 3]];
+        $sort = Sort::only(['id'])->withOrderString('id');
```

Comment:
```suggestion
        $sort = Sort::only(['id']);
```

Should work without `withOrderString('id')`.